code/datums/status_effects/debuffs/drunk.dm

```diff
@@ -57,7 +58,8 @@
 /datum/status_effect/inebriated/proc/set_drunk_value(set_to)
 	if(!isnum(set_to))
 		CRASH("[type] - invalid value passed to set_drunk_value. (Got: [set_to])")
-
+	if(drunk_value >= 51 && iron_liver)
+		return
```

Comment:
This code doesn't actually *cap* at at 51 btw - all it does is block setting to value above 51 - in fact if you're above 51 it means it literally can't decay anymore 

If you want to cap it you should do `if(iron_liver) set_to = min(51, set_to)`

Reply:
gotcha